doc/modules/clustering.rst

```diff
@@ -1128,12 +1128,16 @@ random from :math:`U` falls into class :math:`U_i`. Likewise for :math:`V`:
 
 With :math:`P'(j) = |V_j| / N`. The mutual information (MI) between :math:`U`
 and :math:`V` is calculated by:
-
+ 
```

Comment:
Can you remove this additional space and the one below? Avoiding unnecessary changes like these contributes to make the review process more pleasant for everyone involved.